tests/python/tenseal/tensors/test_ckks_vector.py

```diff
@@ -1,8 +1,12 @@
-import tenseal as ts
-import pytest
-import numpy as np
 import copy
 import pickle
+import math
+import pytest
+
+import numpy as np
+from skimage.util.shape import view_as_windows
```

Comment:
The Bazel build will fail because of missing depends.

To fix that, you can add the line 
```
requirement("scikit-image")
```

to the tests/python/tenseal/BUILD file

Reply:
I was looking for it.
Got it, thanks.

Reply:
So there is no way to have BAZEL fetch that from requirements files?

Reply:
It actually fetches those requirements, but uses them only in the specified libraries/tests. It won't add any additional dependency by default